eole/predict/beam_search.py

```diff
@@ -357,30 +356,34 @@ def advance(self, log_probs, attn):
         self.maybe_update_forbidden_tokens()
 
         if self.return_attention or self._cov_pen:
-            current_attn = attn[self.select_indices]
             if step == 1:
-                self.alive_attn = current_attn
```

Comment:
It leads to an error so it is taken off. The alive attn is only used for the return attn path, so it is separated inside the coverage path.

Reply:
That seems weird. Isn't it needed in `update_finished` and `remove_finished_batches`?